ui/lib/core/addon/components/alert-banner.js

```diff
@@ -33,4 +35,8 @@ export default Component.extend({
   alertType: computed('type', function() {
     return messageTypes([this.get('type')]);
   }),
+
+  secondAlertType: computed('secondIconType', function() {
+    return messageTypes([this.get('secondIconType')]);
```

Comment:
does `messageTypes` error out if no `secondIconType` has been set?

Reply:
Nope.  I tested this, both with not having the secondIconType in the component and also have a blank string for the `secondIconType=""`.  They still work.

Reply:
What about an iconType that isn't valid (like "foo") just to be safe?

Reply:
Good question.  It still loads the `alertBanner`, it just doesn't show a second Icon, which is what I would expect. 
![image](https://user-images.githubusercontent.com/6618863/79472696-63eb4480-7fc1-11ea-912e-848de32157b4.png)